### PR TITLE
Update mingw-w64-mesa to 20.1.7 and add swr driver to MINGW64 package

### PR DESCRIPTION
**Pull request by pal1000**

### CODE REVIEW

**Comment by Biswa96** [comment]
Just asking. Shouldn't it be `-Dc_args=${CFLAGS} -Dcpp_args=${CXXFLAGS}` as those are configured in `makepkg.conf` by user? Also it looks like the backslashes were removed accidentally.

File: mingw-w64-mesa/PKGBUILD [hunk 1]
```diff
@@ -74,23 +81,30 @@ prepare() {
...
-  -Dc_link_args='-s' \
-  -Dcpp_link_args='-s' \
-  -Dshared-glapi=true \
-  -Dgles1=true \
-  -Dgles2=true \
-  -Dllvm=true \
-  -Dosmesa=gallium \
-  -Dbuild-tests=true
+  buildconf="${MINGW_PREFIX}/bin/meson ./build/windows-${_mach}
+  --prefix=${MINGW_PREFIX}
```
Comment (lazka) [inline mingw-w64-mesa/PKGBUILD:85]: missing all the backslashes?

**Comment by Biswa96** [comment]
Just asking. Are those `*_args` option required? Aren't those flags configured using `makepkg.conf`?

File: mingw-w64-mesa/PKGBUILD (hunk 1 above)
Comment (pal1000) [reply mingw-w64-mesa/PKGBUILD:85]: I had to drop them to add that conditional string concatenation for swr. I have to figure something out, hopefully without relying on CFLAGS, CXXFLAGS and LDFLAGS.

**Comment by pal1000** [comment]
> 
> 
> Just asking. Are those `*_args` option required? Aren't those flags configured using `makepkg.conf`?

Some of them provide performance benefit around 2-5%

**Comment by pal1000** [comment]
Found a solution. I moved the args at the end of build config command which allowed me to put back the backslashes for them.

**Pull request merged**